code/modules/mob/living/simple_animal/abnormality/waw/ebony_queen.dm

```diff
@@ -69,7 +69,7 @@
 
 /mob/living/simple_animal/hostile/abnormality/ebony_queen/update_icon_state()
 	if(status_flags & GODMODE) // Not breaching
-		icon_state = initial(icon)
+		icon_state = initial(icon_state)
```

Comment:
Might want to remove update_icon_state override altogether.

Reply:
I forgot why this was even needed.